stdlib/public/Platform/winsdk.modulemap

```diff
@@ -129,6 +129,12 @@ module WinSDK [system] {
       header "timezoneapi.h"
       export *
     }
+    
+    // api-ms-win-core-usb-l1-1-0.dll
+    module usb {
+      header "winusb.h"
+      export *
+    }
```

Comment:
This isn't correct really.  `winusb.h` is not part of the API Set contracts.  See https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-api-list for more details.  This should be pulled out of `core` because it is not part of core os (onecore).  Additionally, it should add the import library for autolinking resolution (`wibusb.lib`).

Reply:
Updated